app/controllers/ReviewController.scala

```diff
@@ -47,8 +49,17 @@ class ReviewController @Inject()(
         BadRequest(views.html.review(reviewDetails.consignment, reviewDetails.fileChecks, formWithErrors))
       )
     }
+
     val successFunction: ReviewData => Future[Result] = { formData: ReviewData =>
-      Future.apply(Redirect(routes.TransferConfirmationController.index()))
+      val appSyncClient = client.graphqlClient
```

Comment:
Probably should not use the val name "appSyncClient" as not using app sync anymore

Reply:
Fixed.